src/Microsoft.ML.ImageAnalytics/ImagePixelExtractor.cs

```diff
@@ -44,8 +44,11 @@ namespace Microsoft.ML.ImageAnalytics
     public sealed class ImagePixelExtractingTransformer : OneToOneTransformerBase
     {
         [BestFriend]
-        internal class Column : OneToOneColumn
+        internal class Options : TransformInputBase
         {
+            [Argument(ArgumentType.Multiple | ArgumentType.Required, HelpText = "New column definition(s) (optional form: name:src)", Name = "Column", ShortName = "col", SortOrder = 1)]
+            public ImagePixelExtractingEstimator.ColumnOptions[] Columns;
+
             [Argument(ArgumentType.AtMostOnce, HelpText = "Whether to use alpha channel", ShortName = "alpha")]
             public bool? UseAlpha;
```

Comment:
>UseAlpha [](start = 25, length = 8)

These are a little funky. In entry-point land, and command line land, there would be the arguments to the transform, and then the arguments to each column that could potentially override it.